packages/babel-generator/src/printer.ts

```diff
@@ -72,6 +72,9 @@ class Printer {
   constructor(format: Format, map: SourceMap) {
     this.format = format;
     this._buf = new Buffer(map);
+
+    this._indentChar = format.indent.style.charCodeAt(0);
+    this._indentRepeat = format.indent.style.length;
```

Comment:
This technically breaks `indent.style === " \t"`, but no one will notice :shushing_face: